src/__experimental__/components/field-help/field-help.style.js

```diff
@@ -9,20 +9,20 @@ const FieldHelpStyle = styled.span`
   white-space: pre-wrap;
   width: 100%;
 
-  ${({ labelInline, inputWidth }) => labelInline && css`
+  ${({ labelInline, labelWidth }) => labelInline && css`
```

Comment:
field help is a component directly below the input and it should have the same width as that input